beacon_node/beacon_chain/src/validator_monitor.rs

```diff
@@ -1344,131 +1344,131 @@ impl<T: EthSpec> ValidatorMonitor<T> {
             for (_, validator) in self.validators.iter() {
                 let id = &validator.id;
                 let summaries = validator.summaries.read();
+                let default_summary = EpochSummary::default();
```

Comment:
Perhaps we could boost this line above the `for` loop on 1344 to reduce allocations.